src/client/traits/index.coffee

```diff
@@ -38,6 +38,7 @@ module.exports = (api)->
         # });
       _.map name, (value, key)-> new Trait(api, key, value)
       Hull.emit('hull.traits',name)
+      Hull.emit('hull.identify',name)
```

Comment:
does it mean we start to emit two things on `Hull.traits()` call?

Reply:
we emit the same message on 2 channels

Reply:
does it mean we can start producing higher volumes of incoming events?

Reply:
this is the client-side Hull Event Emitter:  https://www.hull.io/docs/reference/hull_js/#event-emitter

Not related to tracking calls.